app/scripts/app.js

```diff
@@ -46,4 +46,5 @@ app.config(function ($routeProvider) {
       redirectTo: '/'
     });
 })
-  .constant('defaultFolder', '2016-01-14');
+  .constant('defaultFolder', '2016-01-14')
```

Comment:
It is not constant or it should not be constant. We have discussed this and decided that default normative dataset should not be hardcoded (i.e constant) but should be set in release.json file.
Also this change does not mach the commit message and should not be here.